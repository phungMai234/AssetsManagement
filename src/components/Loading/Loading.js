import React from 'react';

import styled from 'styled-components';

const LoadingWrapper = styled.div`
  width: 100%;
  align-items: center;
  display: flex;
  justify-content: center;
  padding: 5px;
`;

const Loading = () => {
  return (
    <LoadingWrapper>
      <div className="spinner-border" role="status">
        <span className="sr-only">Loading...</span>
      </div>
    </LoadingWrapper>
  );
};

export default Loading;
