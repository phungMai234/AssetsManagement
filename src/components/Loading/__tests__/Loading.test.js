import React from 'react';

import { render } from '@testing-library/react';

import Loading from '../Loading';

test('should render correctly', () => {
  const container = render(<Loading />);
  expect(container.container).toMatchSnapshot();
});
