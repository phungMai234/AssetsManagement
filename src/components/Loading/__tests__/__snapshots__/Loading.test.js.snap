// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <div
    class="sc-bdfBwQ bmhzKe"
  >
    <div
      class="spinner-border"
      role="status"
    >
      <span
        class="sr-only"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;
